### PR TITLE
Fix #910

### LINKED ISSUE #910: Hamburger icon is not aligned right on using container as first child of navbar
### Overview of the problem

Tried this fix #896 which brought out this issue.  The hamburger menu icon which appears in the mobile view is not aligned to the right when using a container inside a navbar and it looks odd.

Check this https://codepen.io/connectkushal/pen/prodge?editors=1100

This is about the Bulma **CSS framework**
 I'm using Bulma **version** [0.4.4]

I am sure this issue is **not a duplicate** 

### Steps to Reproduce

1. Copy the navbar example code as in docs
2. Wrap everything in the parent nav in a container div
3. See mobile view

### PR DESCRIPTION
**Pull request by Come2Daddy**
Fix #910

### CODE REVIEW

File: sass/components/navbar.sass [hunk 1]
```diff
@@ -112,6 +112,8 @@ a.navbar-item,
   margin: 0.5rem 0
 
 +touch
+  .container
```
Comment (jgthms) [inline sass/components/navbar.sass]: This needs to be `.navbar > .container`

**Comment by joshmanders** [comment]
ping @Come2Daddy

**Comment by connectkushal** [comment]
double ping @Come2Daddy

**Comment by jgthms** [comment]
That's the wrong solution. Should be `display: block`.

**Pull request closed without merging**